numpy/distutils/fcompiler/gnu.py

```diff
@@ -315,6 +315,12 @@ def version_match(self, version_string):
     module_dir_switch = '-J'
     module_include_switch = '-I'
 
+    if sys.platform[:3] == 'aix':
+        executables['linker_so'].append('-lpthread')
```

Comment:
Is this needed for all Fortran code, or only for (e.g.) BLAS?

The only other place we have this is for MKL BLAS (defined in `distutils/system_info.py`).

Reply:
Yes, this is needed for Fortran code. I added little more info about this here --> https://github.com/numpy/numpy/issues/12396#issuecomment-439348349